contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -470,6 +488,22 @@ def _execute(self, track_name_filter: str, expect_exists: bool = True):
                                  'Track "{}" has no callstacks pane'.format(track.name))
 
 
+class SelectAllCallstacks(CaptureWindowE2ETestCaseBase):
+
+    def _execute(self, track_name_filter: str):
+        tracks = self._find_tracks(track_name_filter)
+        self.expect_true(
+            len(tracks) == 1, 'Found exactly one track matching "{}"'.format(track_name_filter))
+        track = Track(tracks[0])
+        self.expect_true(track.callstacks is not None,
+                         'Track "{}" has callstacks pane'.format(track.name))
+
+        logging.info('Selecting all visible callstacks in track "{}"'.format(track.name))
+        callstacks = track.callstacks
+        rect = callstacks.rectangle()
+        callstacks.drag_mouse_input(src=(rect.left, rect.top + 5), dst=(rect.right, rect.top + 5))
```

Comment:
It would be great to verify the expected effect of callstack selection after this. Would it make sense to check if the "Selection" tabs are enabled as a minimal test? A more detailed test could also verify if the # of samples in all selection-tabs sum up to an expected number, but this is probably out of scope for this test.